docs_src/tutorial_2_configuration.py

```diff
@@ -0,0 +1,26 @@
+from corva import Api, Corva, Event, State
+
+app = Corva(
+    # 1 api params
+    api_url='api.localhost',
```

Comment:
Are these params pulled from env variables by default?

Reply:
yes, they will be provided by default, with possibility to override

also some unique settings for each app type will be moved to `stream`, `task` and `scheduled` decorators